content/applications/finance/payment_acquirers/paypal.rst

```diff
@@ -113,6 +87,48 @@ Your Paypal account is ready!
      `link <https://sandbox.paypal.com/cgi-bin/customerprofileweb?cmd=_profile-language-encoding>`_
      to configure the encoding format in a test environment.
 
+Settings in Odoo
+================
+
+.. seealso::
+   :ref:`payment_acquirers/add_new`
+
+Credentials
+-----------
+
+Odoo needs your **API Credentials** to connect with your PayPal account. To do so, go to
+:menuselection:`Accounting --> Configuration --> Payment Acquirers` and :guilabel:`Activate` PayPal.
+Then, enter your PayPal account credentials in the :guilabel:`Credentials` tab:
+
+- :guilabel:`Email`: the login email address in Paypal;
+- :guilabel:`PDT Identity Token`: the key used to verify the authenticity of transactions;
+- :guilabel:`Use IPN`: enable, as it is :ref:`required <paypal/auto-return>` for PayPal to work
+  properly in Odoo.
+
+To set the :guilabel:`PDT Identity Token`, switch to :ref:`developer mode <developer-mode>` and
+retrieve the token by following the configuration steps at :ref:`paypal/pdt`.
```

Comment:
In addition, or alternative, I suggest following what was done in the latest payment provider docs where we recommend saving the generated keys for later: https://www.odoo.com/documentation/master/applications/finance/payment_providers/amazon_payment_services.html#configuration-on-aps-dashboard